src/components/Rating/readme.md

```diff
@@ -84,6 +84,93 @@ class SimpleRating extends React.Component {
 </div>
 ```
 
+##### Rating readOnly
+
+```js
+import React from 'react';
+import { Card, ButtonGroup, ButtonIcon, Rating } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+const cardStyles = {
+    width: 300,
+};
+
+const imageStyles = {
+    borderTopLeftRadius: '0.875rem',
+    borderTopRightRadius: '0.875rem',
+    height: 170,
+    width: '100%',
+    backgroundImage: 'url(images/illustrations/Illustration-rainbow-4.svg)',
+    backgroundSize: 'cover',
+};
+
+const StyledHeader = styled.h3.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: ${props => props.text.main};
+`;
+
+const StyledText = styled.h3.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: ${props => props.text.label};
+`;
+
+class SimpleRating extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: '3.5',
+        };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    handleOnChange(event) {
+        console.log(' changed! ');
+        return this.setState({ value: event.target.value });
+    }
+
+    render() {
+        return <Rating value={this.state.value} onChange={this.handleOnChange} readOnly />;
+    }
+}
```

Comment:
Since this example is `readOnly` we don't need to use a state, use directly the `Rating` component inside the Card